### PR TITLE
gobo 20.01 (new formula)

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/gobo.rb [hunk 1]
```diff
@@ -0,0 +1,94 @@
+class Gobo < Formula
+  desc "Free and portable Eiffel tools and libraries"
+  homepage "http://www.gobosoft.com/"
+  url "https://downloads.sourceforge.net/project/gobo-eiffel/gobo-eiffel/20.01/gobo2001-src.tar.gz"
```
Comment (SMillerDev) [inline Formula/gobo.rb]: What's the license for this?

Comment (pthariensflame) [reply Formula/gobo.rb]: MIT:  http://www.gobosoft.com/eiffel/gobo/license.html

**Comment by pthariensflame** [comment]
@SMillerDev Any updates needed here?

**Comment by SMillerDev** [comment]
Thanks @pthariensflame! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**